docs/serverless/investigate/cases-settings.mdx

```diff
@@ -40,7 +41,7 @@ To create a new connector
 
 1. From the **Incident management system** list, select **Add new connector**.
 
-1. Select the system to send cases to: **((sn))**, **((jira))**, **((ibm-r))**, **((swimlane))**, or **((webhook-cm))**.
+1. Select the system to send cases to: **((sn))**, **((jira))**, **((ibm-r))**, **((swimlane))**, **TheHive**, or **((webhook-cm))**.
    ![Shows the page for creating connectors](../images/cases-settings/security-cases-connectors.png)
```

Comment:
Could we omit this screenshot? It's not included in the ESS/AsciiDoc docs, and doesn't seem like essential information for the user.

Reply:
One less screenshot to maintain works for me!

Reply:
Fixed in 98fcda77c6ef4a71c58a90fb75b7eccfb300a6e1